pkg/services/auth/auth_token.go

```diff
@@ -404,6 +414,36 @@ func (s *UserAuthTokenService) rotatedAfterParam() int64 {
 	return getTime().Add(-s.Cfg.LoginMaxInactiveLifetime).Unix()
 }
 
+func (s *UserAuthTokenService) revokeOldUserTokens(ctx context.Context, userId int64, maxConcurrentSessions int) error {
+	return s.SQLStore.WithDbSession(ctx, func(dbSession *sqlstore.DBSession) error {
+		sql := `
+		DELETE
+		FROM user_auth_token
+		WHERE id NOT IN
+			(
+			  SELECT id
+			  FROM user_auth_token
+			  WHERE user_id = ?
```

Comment:
We don't have an index for `user_id` in https://github.com/grafana/grafana/blob/b0df7280be60be815078e572f5975a14521695bf/pkg/services/sqlstore/migrations/user_auth_token_mig.go#L8-L27

@marefr wdyt, should we add an index here or is there a better way to go about this?

Reply:
Interesting one! Let's wait for @marefr point of view, but it might have sense since filtering by `user_id` will still be required, I guess 🤔  

Thanks!